Examples/Examples/TableTopExampleView.swift

```diff
@@ -16,7 +16,6 @@ import ArcGIS
 import ArcGISToolkit
 import SwiftUI
 
-@available(macCatalyst, unavailable)
```

Comment:
I think you can make the same change for `WorldScaleExampleView`  & `FlyoverExampleView` then change both target membership for those files to just iOS.

Reply:
Sure, I'll make those changes.